src/test/suite/format/format.test.ts

```diff
@@ -20,6 +20,7 @@ const defaultOptions: vscode.FormattingOptions = {
 const formatTests: FormatTest[] = [
     { filenameRoot: '25-multiline-string' },
     { filenameRoot: '40-command-inside-text' },
+    { filenameRoot: '55-sharp-directive' },
```

Comment:
We should test this both with and without the config value set. Kind of like how we have tests for both "useTabs" and "useSpaces", if that makes sense

Reply:
Done.